externals/kyuubi-jdbc-engine/src/test/scala/org/apache/kyuubi/engine/jdbc/mysql/WithMysqlContainer.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kyuubi.engine.jdbc.mysql
+
+import com.dimafeng.testcontainers.{GenericContainer, SingleContainer}
+import org.testcontainers.containers.wait.strategy.Wait
+
+import org.apache.kyuubi.engine.jdbc.WithJdbcServerContainer
+
+trait WithMysqlContainer extends WithJdbcServerContainer {
```

Comment:
Better to use testcontainers for Mysql instead.
https://github.com/testcontainers/testcontainers-scala/blob/master/docs/src/main/tut/usage.md#mysql

Reply:
> Better to use testcontainers for Mysql instead. https://github.com/testcontainers/testcontainers-scala/blob/master/docs/src/main/tut/usage.md#mysql

We are currently using testcontainers-scala-scalatest, which already meets our needs. To use Mysql testcontainers, we need to add the `testcontainers-scala-mysql` dependency. Is it necessary?

Reply:
It's alright to introduce test scope dependency for testing different datasource provider. In this way, it aovids redundant wrapping lower level testcontainers api. Testconainers for kafka was included and for es is being included in another pending pr.

Reply:
> It's alright to introduce test scope dependency for testing different datasource provider. In this way, it aovids redundant wrapping lower level testcontainers api. Testconainers for kafka was included and for es is being included in another pending pr.

OK.